.github/workflows/build-and-test-arm.yml

```diff
@@ -6,9 +6,6 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+*"
   merge_group:
   pull_request:
-    types: [opened, synchronize, reopened, labeled, unlabeled]
-    branches:
```

Comment:
do we not want to keep this restriction for running it on PRs against main?

Reply:
it's running on all PRs against main already.